webknossos-datastore/app/views/datastoreFrontpage.scala.html

```diff
@@ -0,0 +1,26 @@
+@( kind: String, name: String, uri: String, wKUri: String, healthPath: String )
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+  <title>webKnossos @(kind)</title>
+  <meta name="robot" content="noindex" />
+  <link rel="shortcut icon" type="image/png" href="https://static.webknossos.org/images/oxalis.svg" />
+  <style>
+    * {font-family: "Monospaced Number", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial,
+    sans-serif;
+    text-align: center}
+    img {display: block; margin: auto; padding: 3em 2em 0.5em 2em}
+    a {color: black; text-decoration: none; font-size: 2em}
+    #hint {color: #777; margin-top: 4em}
+</style>
+</head>
+<body>
+<a href="@(wKUri)" id="wKlink">
+  <img src="https://static.webknossos.org/images/oxalis.svg" alt="" />
+  <span>Go to webKnossos</span>
+</a>
+<p id="hint">This is the webKnossos @(kind) “@(name)”.</p>
```

Comment:
Maybe use JS for the healthcheck?
```suggestion
<p id="hint">This is the webKnossos @(kind) “@(name)”. Healthcheck: <span id="healthcheck">Loading...</span></p>
<script>
fetch("/data/health")
  .then((r) => r.text())
  .then((status) => {
    document.getElementById("healthcheck").innerHTML = status;
  })
  .catch((err) => {
    document.getElementById("healthcheck").innerHTML = "Error";
  });
</script>
```

Reply:
yeah, good idea :)